tests/test_app/config/Migrations/20150416223600_mark_migrated_test.php

```diff
@@ -2,6 +2,8 @@
 
 use Migrations\AbstractMigration;
 
+// phpcs:disable PSR1.Classes.ClassDeclaration.MissingNamespace
+// phpcs:disable PSR2R.Classes.ClassFileName.NoMatch
```

Comment:
Should we ignore these rules in the rulesets we provide? That would be less noise in generated migrations and would retroactively apply to existing migrations.

Reply:
How would that look like?
In custom phpcs file per repo?

Reply:
Required rules can be made to ignore `config/Migrations` path similar to this https://github.com/cakephp/cakephp-codesniffer/blob/master/CakePHP/ruleset.xml#L121-L124

Reply:
I think we wont have this issue solved for years. Putting the burden on the repos themselves might increase the workload long term.
But either way works for me.

Reply:
I am suggesting handling this by updating our codesniffer's rulesset file itself, so other repos won't have to do anything extra.

Reply:
For non-cake users (using phinx directly), I opened https://github.com/cakephp/phinx/issues/1938
There might still be a different strategy needed here, then.

Reply:
Also, not everyone is using the code sniffer repo on project level, so in the end there will still be often something to adjust on top.
But still no hard argument against the sniffer adjustment.